### PR TITLE
Fixing invalid syntax in quickstart docs page

### PR DESCRIPTION
**Pull request by ByteWolfCoding**
## What type of change does this PR perform?

- [x] Info or documentation change (Non-breaking change that updates dependencies, info files or online documentation)

## Summary

Fixed invalid syntax on the quickstart page.

## Summary of Changes

Fixed the invalid TS and JS snippets on the quickstart page.

## Does this PR create new warnings?

_No._

## Detailed Changelog

_Not present for website/docs changes._

_None._

## Linked issues or PRs
_No linked issues._

### CODE REVIEW

File: src/docs/0.9.2/quickstart.md [hunk 1]
```diff
@@ -149,57 +149,53 @@ Kipper is shipped as a commonjs module and can therefore be imported using `requ
...
 });
 ```
 
 Using TypeScript:
 
 ```kipper
-import * as ts from "typescript";
 import { promises as fs } from "fs";
 import * as kipper from "@kipper/core";
+import * as kipperJS from "@kipper/target-js";
```
Comment (Luna-Klatzer) [inline src/docs/0.9.2/quickstart.md]: Here `typescript` is still required, as in that version there was no `@kipper/target-js` package. So just leave the example here as it is.

File: src/docs/latest/quickstart.md [hunk 2]
```diff
@@ -208,25 +208,25 @@ Using JavaScript:
...
+    // Define your own logger and compiler, which will handle the compilation
+    const logger = new kipper.KipperLogger((level, msg) => {
+        console.log(`[${level}] ${msg}`);
+    });
+    const compiler = new kipper.KipperCompiler(logger);
+
+    // Compile the code string or stream
+    let result = await compiler.compile(fileContent, {
+        /* Config - Target must always be specified */
+        target: new kipperJS.TargetJS,
```
Comment (Luna-Klatzer) [inline src/docs/latest/quickstart.md]: Missing `()` in the `new` expression.

So `new kipperJS.TargetJS()` is correct.

File: src/docs/next/quickstart.md [hunk 3]
```diff
@@ -208,25 +208,25 @@ Using JavaScript:
...
+    // Define your own logger and compiler, which will handle the compilation
+    const logger = new kipper.KipperLogger((level, msg) => {
+        console.log(`[${level}] ${msg}`);
+    });
+    const compiler = new kipper.KipperCompiler(logger);
+
+    // Compile the code string or stream
+    let result = await compiler.compile(fileContent, {
+        /* Config - Target must always be specified */
+        target: new kipperJS.TargetJS,
```
Comment (Luna-Klatzer) [inline src/docs/next/quickstart.md]: Missing `()` in the `new` expression.

So `new kipperJS.TargetJS()` is correct.

File: src/docs/next/quickstart.md [hunk 4]
```diff
@@ -239,20 +239,20 @@ import * as kipperJS from "@kipper/target-js";
...
```
Comment (Luna-Klatzer) [inline src/docs/next/quickstart.md]: Missing `()` in the `new` expression.

So `new kipperJS.TargetJS()` is correct.

File: src/docs/0.9.2/quickstart.md (hunk 1 above)
Comment (Luna-Klatzer) [reply src/docs/0.9.2/quickstart.md]: Didn't mention that before sorry!

**Review by Luna-Klatzer — CHANGES_REQUESTED** [review]
Minor edits still required.

**Review by Luna-Klatzer — APPROVED** [review]

File: src/docs/latest/quickstart.md (hunk 2 above)
Comment (ByteWolfCoding) [reply src/docs/latest/quickstart.md:224]: Is that ment for the JavaScript or the Typescript example? Or both?

File: src/docs/latest/quickstart.md [hunk 5]
```diff
@@ -239,20 +239,20 @@ import * as kipperJS from "@kipper/target-js";
...
```
Comment (Luna-Klatzer) [inline src/docs/latest/quickstart.md]: Still `()` missing.

File: src/docs/0.9.2/quickstart.md [hunk 6]
```diff
@@ -149,29 +149,27 @@ Kipper is shipped as a commonjs module and can therefore be imported using `requ
 Using JavaScript:
 
 ```kipper
-const ts = require("typescript");
 const fs = require("fs").promises;
 const kipper = require("@kipper/core");
+const kipperJS = require("@kipper/target-js");
```
Comment (Luna-Klatzer) [inline src/docs/0.9.2/quickstart.md]: This should not be changed.

Like I said, this should remain like it was, so like:
```
const ts = require("typescript");
const fs = require("fs").promises;
const kipper = require("@kipper/core");
```

File: src/docs/0.9.2/quickstart.md [hunk 7]
```diff
@@ -149,29 +149,27 @@ Kipper is shipped as a commonjs module and can therefore be imported using `requ
...
+
+    // Compile the code string or stream
+    let result = await compiler.compile(fileContent, {
+        /* Config - Target must always be specified */
+        target: new kipperJS.TargetJS,
+    });
+    let jsCode = result.write();
+
+    // Running the Kipper program
+    eval(jsCode);
```
Comment (Luna-Klatzer) [inline src/docs/0.9.2/quickstart.md]: This change must be reverted.

**Review by Luna-Klatzer — CHANGES_REQUESTED** [review]

File: src/docs/next/quickstart.md (hunk 4 above)
Comment (Luna-Klatzer) [inline src/docs/next/quickstart.md]: Well, one is still left

**Review by Luna-Klatzer — CHANGES_REQUESTED** [review]

**Review by Luna-Klatzer — APPROVED** [review]

**Pull request merged**